pkg/auth/authorizer/interfaces.go

```diff
@@ -56,12 +59,20 @@ func (f AuthorizerFunc) Authorize(a Attributes) error {
 	return f(a)
 }
 
+type MetaResource int
+
+const (
+	ApiVersions MetaResource = 1 << iota
```

Comment:
I would recommend against enumerating constants for policy like this.  It makes you sensitive to any refactoring order change and precludes reasonable extension by any downstream component.  String constants will avoid those problems.